src/components/App.js

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../styles/App.scss';
 import '../styles/Button.scss';
 import '../styles/ButtonPanel.scss';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
+import calculate from '../logic/calculate';
 
 function App() {
+  const [total, setTotal] = useState(null);
+  const [next, setNext] = useState(null);
+  const [operation, setOperation] = useState(null);
+
+  const handleClick = buttonName => {
+    let data = calculate({ total, next, operation }, buttonName);
```

Comment:
The thing is we don't need `let` here. Our linter probably isn't working as we hope it should. Let's use `const` here and let's make sure ESLint is working properly in this project.